gofish/snapchat/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Log in | Snapchat</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="https://static.snapchat.com/accounts-ui/static/1730142817/images/favicon/favicon.png" type="image/png">
```

Comment:
Don't hotlink to external images, download this one and include it in your PR

Reply:
Alright, I got rid of both Hotlinks